test_parser.py

```diff
@@ -43,7 +45,7 @@ def test_parser(zone, data_type, target_datetime):
     start = time.time()
 
     parser = PARSER_KEY_TO_DICT[data_type][zone]
-    if data_type in ['exchange', 'exchangeForecast']:
+    if data_type in ['exchange', 'exchangzoneeForecast']:
```

Comment:
I think this shouldn't be changed right?

Reply:
Indeed, my apologies for letting that go through in the PR